fhir2dataset/fhirpath.py

```diff
@@ -0,0 +1,112 @@
+"""set of functions allowing to use the javascript coded library on the repository https://github.com/HL7/fhirpath.js
+"""  # noqa
+import json
+import logging
+from subprocess import Popen, PIPE
+
+from fhir2dataset.timer import timing
+
+logger = logging.getLogger(__name__)
+
+wrapper = """
+(function run(globals) {
+    try {
+        var result = (%(func)s).apply(globals, %(args)s);
```

Comment:
You should use let :)
var is the old method

Reply:
It's changed